README.md

```diff
@@ -2,197 +2,74 @@
 
-* Lnd mode **"remote"**
-  + Connect to a remote `lnd` instance, start the rest (the UI, `loop`, 
-  `faraday`) in the same process.
-  + [Please read the `lnd` **remote** mode configuration guide here.](doc/config-lnd-remote.md)
-  + This is the default mode that is used if the `--lnd-mode=` command line
-    or `lnd-mode=` configuration option is not set explicitly.
+To upgrade from v0.1.1-alpha or earlier simply create a `lit.conf` file in your LiT directory. The default location LiT uses depends on your operating system:
+- MacOS: `~/Library/Application Support/Lit/lit.conf`
+- Linux: `~/.lit/lit.conf`
+- Windows: `~/AppData/Roaming/Lit/lit.conf`
 
-* Lnd mode **"integrated"**
-  + Start everything (the UI, `lnd`, `loop`, `faraday`) in one single process.
-  + [Please read the `lnd` **integrated** mode configuration guide here.](doc/config-lnd-integrated.md)
+Move all the configuration settings specific to LiT from `lnd.conf` to `lit.conf` and remove the `lnd.` prefix from the configuration settings in `lnd.conf`. Note that any section headers (`[ Example ]`) in `lit.conf` should be removed or changed to comments (`# Example`). 
 
-### Troubleshooting
-
-If you have trouble running your node, please first check the logs for warnings or errors.
-If there are errors relating to one of the embedded servers, then you should open an issue
-in their respective GitHub repos ([lnd](https://github.com/lightningnetwork/lnd/issues),
-[loop](https://github.com/lightninglabs/loop/issues),
-[faraday](https://github.com/lightninglabs/faraday/issues). If the issue is related to the
-web app, then you should open an
-[issue](https://github.com/lightninglabs/lightning-terminal/issues) here in this repo.
-
-#### Server
-
-Server-side logs are stored in the directory specified by `lnd.lnddir` in your
-configuration. Inside, there is a `logs` dir containing the log files in subdirectories.
-Be sure to set `lnd.debuglevel=debug` in your configuration to see the most verbose
-logging information.
+## Usage
+Read the [walkthrough](doc/WALKTHROUGH.md) document to learn more about how to use LiT.
 
-#### Browser
+## Build from source
+If you’d prefer to compile from source code please follow [these instructions](./doc/compile.md).
 
-Client-side logs are disabled by default in production builds. Logging can be turned on by
-adding a couple keys to your browser's `localStorage`. Simply run these two JS statements
-in you browser's DevTools console then refresh the page:
+## Compatibility
 
-```
-localStorage.setItem('debug', '*'); localStorage.setItem('debug-level', 'debug');
-```
+Lightning Terminal is backwards compatible with LND back to version v0.11.0-beta
 
-The value for `debug` is a namespace filter which determines which portions of the app to
-display logs for. The namespaces currently used by the app are as follows:
+| LiT              | LND          |
+| ---------------- | ------------ |
+| **v0.2.0-alpha** | v0.11.0-beta |
```

Comment:
@guggero can correct me if I'm wrong but I believe `v0.2.0-alpha` requires `lnd` `v0.11.1-beta`

Reply:
On the code/dependency level, yes. But on the RPC level, to connect to a remote node, `v0.11.0-beta` is correct (this isn't even the requirement of LiT itself but faraday).